prereq-check.sh

```diff
@@ -959,6 +959,39 @@ function check_os() (
         fi
     }
 
+    function check_thp_enabled() {
+        # https://docs.cloudera.com/documentation/enterprise/latest/topics/cdh_admin_performance.html#cdh_performance__section_hw3_sdf_jq
+        local file
+        file=$(find /sys/kernel/mm/ -type d -name '*transparent_hugepage')/enabled
+        if [ -f "$file" ]; then
+            local msg="System: $file should be disabled"
+            if grep -F -q "[never]" "$file"; then
+                state "$msg" 0
+            else
+                state "$msg. Actual: $(awk '{print $1}' "$file" | sed -e 's/\[//' -e 's/\]//')" 1
+            fi
+        else
+            state "System: /sys/kernel/mm/*transparent_hugepage not found. Check skipped" 2
+        fi
+    }
+
+    function check_thp_grub() {
+        # If your cluster hosts are running RHEL/CentOS 7.x, modify the GRUB configuration to disable THP
+        # https://docs.cloudera.com/documentation/enterprise/latest/topics/cdh_admin_performance.html#cdh_performance__section_hw3_sdf_jq
+        local file
+        file=$(find /etc/default/grub)
+        if [ -f "$file" ]; then
```

Comment:
Because this is a simple filename, we don't need the `find` command. 

We only need:
`if [ -f "/etc/default/grub" ]; then`